ansible/README.md

```diff
@@ -15,36 +14,86 @@ You will need your public key deployed to the host for ssh access
 
 You may need to connect to the host once with ssh to make sure it's in your list of known hosts
 
+If you install [casey/just](https://github.com/casey/just), then some tasks
+will be simplified.
+
 ## Before Running
 
-Create secret files under secrets-[env]:
+**Setup the Ansible Vault Passphrase**
+There are passwords, secret keys, and other sensitive information required for everything
+to work. These secrets are included in the repo and they are encrypted using
+ansible-vault.  To run ansible-successfully, you will need the passphrase to
+decrypt them.  Ask a team member. Then, run:
+
+```console
+$ echo -n "passphrase" >> .passworf
+```
 
-* 1nfinite.pem (The pem obtained from auth0)
-* web-portal.env (env file with the required environment values for the web-portal service)
-* api.env (env file with the required environment values for the api service)
+*Alternatively:*
+
+```console
+$ just cache-pass
+```
 
 ## Running
 
-`ansible-playbook ./base_playbook.yml`
+### Common Task: Site Deployment
+
+First, deploy our code: `ansible-playbook -l staging deploy_site_playbook.yml`
+* alternative: `just deploy staging`
+
+Next, restart containers.
 
-## First Time Setup
+```
+$ ssh infinite@infinite.industries
+
+prod $ cd docker-files
+prod $ docker-compose up -d
+prod $ sudo systemctl restart nginx
+```
+
+### Task: First Time Setup
 
 **These steps only needs to happen once**
 
-After running ansible you will need to ssh into the machine and run:
+1. Add the IP address and other info the appropriate section of the `hosts`
```

Comment:
Does this step belong at the beginning of this section, it feels a bit out of place coming after instructions for running ansibe. I think the idea before was that these sections were somewhat linear, with this whole "These steps only needs to happen once" being something that you can usually skip